### PR TITLE
MNW-305 Restore account page

### PR DESCRIPTION
**Pull request by loosefer**
image

### CODE REVIEW

File: packages/frontend/src/pages/RestoreAccount/ui/CheckIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const CheckIcon = () => (
```
Comment (NotEternal) [inline packages/frontend/src/pages/RestoreAccount/ui/CheckIcon.tsx:3]: This icon seems pretty common, we can move it to `components/svg/`

File: packages/frontend/src/translations/locales/ru/translation.json [hunk 2]
```diff
@@ -858,12 +858,14 @@
         }
     },
     "reservedForFeesInfo": "Вплоть до ${data} NEAR зарезервировано для покрытия расходов на транзакции.",
-    "restoreAccountModal": {
+    "restoreAccount": {
         "title": "Восстановить аккаунт",
-        "description": "Restoring your account using recovery phrase you will be automatically logged out of all wallets. You will be able to add them again after going through the authorization process using the selected recovery method.",
+        "wallets_one": "Мы нашли {{count}} кошелёк подключённый к Вашему аккаунту",
+        "wallets_few": "Мы нашли {{count}} кошелька подключённых к Вашему аккаунту",
+        "wallets_many": "Мы нашли {{count}} кошельков подключённых к Вашему аккаунту",
```
[the remark this answers is not in the document]
Comment (NotEternal) [reply packages/frontend/src/translations/locales/ru/translation.json:865]: Now I see. It's for **i18next** plurals.

File: packages/frontend/src/pages/RestoreAccount/RestoreAccount.tsx [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+                    <CheckboxCaption>
+                        {t('restoreAccount.checkboxCaption')}
+                    </CheckboxCaption>
+                </Label>
+            </CheckboxWrapper>
+            <Buttons>
+                {(
+                    <FormButton
+                        /*@ts-ignore*/
+                        onClick={console.log}
```
Comment (NotEternal) [inline packages/frontend/src/pages/RestoreAccount/RestoreAccount.tsx:106]: Why is it `console`? We need to show "_enter password_" form, don't we?

**Comment by NotEternal** [comment]
Need to clarify what we should do after accounts deletion. Maybe we can just reload whole page. For now we still see accounts in the user menu. And after clicking on any navigation link the form disappears (I guess it's because of the Ledge error).
It's a general problem. The app has the only way to delete accounts and it reloads the page after such process.

restore

**Comment by loosefer** [comment]
> Need to clarify what we should do after accounts deletion. Maybe we can just reload whole page. For now we still see accounts in the user menu. And after clicking on any navigation link the form disappears (I guess it's because of the Ledge error). It's a general problem. The app has the only way to delete accounts and it reloads the page after such process.
> 
> restore

Create

> Need to clarify what we should do after accounts deletion. Maybe we can just reload whole page. For now we still see accounts in the user menu. And after clicking on any navigation link the form disappears (I guess it's because of the Ledge error). It's a general problem. The app has the only way to delete accounts and it reloads the page after such process.
> 
> restore

https://mnw.atlassian.net/browse/MNW-322

File: packages/frontend/src/pages/RestoreAccount/ui/CheckIcon.tsx (hunk 1 above)
Comment (loosefer) [reply packages/frontend/src/pages/RestoreAccount/ui/CheckIcon.tsx]: fixed

File: packages/frontend/src/pages/RestoreAccount/RestoreAccount.tsx (hunk 3 above)
Comment (loosefer) [reply packages/frontend/src/pages/RestoreAccount/RestoreAccount.tsx]: fixed

**Review by NotEternal — APPROVED** [review]

**Pull request merged**